transformation/plugins/org.jboss.tools.fuse.transformation.editor/src/main/java/org/jboss/tools/fuse/transformation/editor/internal/l10n/messages.properties

```diff
@@ -186,11 +186,13 @@ TransformationEditor_helpTextTarget=Create a new mapping below by dragging a var
 TransformationEditor_invalidPomConfiguration=Unable to recognize project's POM configuration.
 TransformationEditor_invalidTransformationFile=Invalid transformation file: Must be a Dozer configuration file.
 TransformationEditor_messageDialogConfirmation=Are you sure?\n\nAll incomplete mappings will be lost when the editor is closed.
+TransformationEditor_migrationDialogConfirmation=The transformation file must be converted to support the current Camel version.\n\nAre you sure you want to continue?
```

Comment:
I think that we should provide more details to user and it is not true that it is "support the current Camel version", it is for the Data Transformation editor to continue support Camel 2.20- versions:
- what is modified?
- what are the consequences if "want to continue"? if "does not want to continue"?

Reply:
Proposal:
"The transformation file needs to be updated to continue to use Data Transformation editor. A newer Dozer namespace corresponding to Dozer 6.1+/Camel 2.20+ is required. Updated file will continue to work with previous Camel/Dozer version.

Do you want to proceed to automatic update to use the Data Transformation editor?"

for the "Updated file will continue to work with previous Camel/Dozer version.", i'm not sure if we can write it as it is only "based on our current testing":
- it might change in the future (if the Dozer xsd is modified)
- Dozer is saying nothing about this backward-compatibility

Reply:
Regarding your last point, I think we need to rely on the fact that Dozer isn't mentioning anything about the upgrade that breaks backward-compatibility.  I don't see any other alternative than relying on whatever testing we have performed/will perform to ensure this. Future issues we'll deal with when we see future updates to Dozer; there's always the possibility that changes in the future will break things for any software library.